example/lfp/lfp.cpp

```diff
@@ -79,8 +79,7 @@ struct lfp_demo_recipe: public arb::recipe {
         // Apical dendrite, length 490 μm, radius 1 μm, with SWC tag 4.
         tree.append(soma_apex, {0, 0, 10, 1},  {0, 0, 500, 1}, 4);
 
-        decor dec;
-
+        auto dec = arb::decor();
```

Comment:
Out of curiosity: why?

Reply:
Homogeneity with the other examples. There's no semantic difference.